src/box/lua/config/applier/console.lua

```diff
@@ -3,6 +3,8 @@ local console = require('console')
 local log = require('internal.config.utils.log')
 local instance_config = require('internal.config.instance_config')
 
+local old_socket = nil
```

Comment:
I guess it should be dropped to `nil` if the console is stopped. Please, add a test case that reveals a problem if we don't do that.

Reply:
Does `enabled = false` mean that the console is stopped? If yes, then this case is covered by existing tests
(test/config-luatest/console_test.lua:test_basic).
```.lua
-- Disable the console.
local new_config_2 = cbuilder.new(config)
    :set_global_option('console.enabled', false)
    :config()
cluster:reload(new_config_2)
assert_console_closed(cluster, config)
assert_console_closed(cluster, new_config)
assert_console_closed(cluster, new_config_2)
```

Reply:
It seems important to also verify that the console can be enabled back after disabling (without changing the socket path).

Reply:
test added